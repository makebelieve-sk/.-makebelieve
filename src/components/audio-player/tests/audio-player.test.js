import React from 'react';
import renderer from 'react-test-renderer';
// import Enzyme, { shallow } from 'enzyme';
// import Adapter from 'enzyme-adapter-react-16';

import AudioPlayer from '../audio-player';

// Enzyme.configure({ adapter: new Adapter() });

// const setUp = (prop1, prop2, prop3) => shallow(<AudioPlayer onPlayButtonClick={prop1} src={prop2} isPlaying={prop3} />);

// const mock = {
//     src: `src`,
//     onPlayButtonClick: jest.fn(),
//     isPlaying: false
// };

describe(`AudioPlayer component`, () => {
    // let component;

    // beforeEach(() => {
    //     component = setUp(mock.onPlayButtonClick, mock.src, mock.isPlaying);
    // });

    // it(`should contain section wrapper`, () => {
    //     const wrapper = component.find('.wrapper-audioplayer');
    //     expect(wrapper.length).toBe(1);
    // });

    it(`AudioPlayer correctly renders after relaunch`, () => {
        const tree = renderer  
            .create(<AudioPlayer 
                onPlayButtonClick={jest.fn()}
                src={`src`}
                isPlaying={false}
            />)
            .toJSON();

        expect(tree).toMatchSnapshot();
    });
});
