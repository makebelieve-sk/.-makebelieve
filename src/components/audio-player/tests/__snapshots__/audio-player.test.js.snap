// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AudioPlayer component AudioPlayer correctly renders after relaunch 1`] = `
Array [
  <button
    className="track_button track_button-play"
    disabled={true}
    onClick={[Function]}
    type="button"
  />,
  <div
    className="track_status"
  >
    <audio />
  </div>,
]
`;
